src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7534,7 +7622,7 @@ components:
         - $ref: '#/components/schemas/SnapshotBuilderCriteria'
         - type: object
           required:
-            -conceptId
+            - conceptId
```

Comment:
I happened to notice this typo; the missing space here resulted in this field not being required.